sections/experiments/experiments-chapter.tex

```diff
@@ -498,24 +505,186 @@ \subsubsection*{Ablation study for non-contextual recommendation}
+            (180,0.07608)
+            (200,0.07432)
+        };
+        \legend{Sideinfo + context,Sideinfo,Context,Nothing}
+        \end{axis}
+    \end{tikzpicture}
+    \caption{The performance on HR@20 of CSGCN-ADJ on the Yelp-NC dataset with different types of input.}
+    \label{subfig:ablation_study_context_specific_1}
+\end{subfigure}
+\hspace{0.1\textwidth}
+\begin{subfigure}[t]{0.4\textwidth}
+    \captionsetup{justification=centering,margin={1.4cm,-1.3cm}}
+    \begin{tikzpicture}
+    \begin{axis}[
+            xlabel=Epoch,
+            ylabel=HR@20,
+            height=8cm,
+            width=7cm,
+            grid=major,
+            legend style={at={(0.5,-0.20)},
+        anchor=north,legend columns=2}]
+            
+        \addplot plot coordinates{
+            (20,0.04881)
+            (40,0.05145)
+            (60,0.05409)
+            (80,0.05409)
+            (100,0.05629)
+            (120,0.05585)
+            (140,0.05805)
+            (160,0.06245)
+            (180,0.06201)
+            (200,0.05937)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04266)
+            (40,0.04749)
+            (60,0.04661)
+            (80,0.04925)
+            (100,0.05145)
+            (120,0.04925)
+            (140,0.05277)
+            (160,0.05453)
+            (180,0.05673)
+            (200,0.05717)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04969)
+            (40,0.04705)
+            (60,0.05145)
+            (80,0.05453)
+            (100,0.05585)
+            (120,0.05453)
+            (140,0.05893)
+            (160,0.06069)
+            (180,0.06157)
+            (200,0.06157)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04749)
+            (40,0.04661)
+            (60,0.04749)
+            (80,0.05101)
+            (100,0.05013)
+            (120,0.05057)
+            (140,0.05365)
+            (160,0.05145)
+            (180,0.05277)
+            (200,0.05497)
+        };
+        \legend{Sideinfo + context,Sideinfo,Context,Nothing}
+        \end{axis}
+    \end{tikzpicture}
+    \caption{The performance on HR@20 of CSGCN-IS on the Yelp-NC dataset with different types of input.}
+    \label{subfig:ablation_study_context_specific_2}
+    \end{subfigure}
+    \caption{Ablation study for the input of CSGCN-ADJ and CSGCN-IS on Yelp-NC.}
+\end{figure*}
+
+\Cref{subfig:ablation_study_context_specific_1} shows the HR@20 for CSGCN-ADJ on YelpNC with different combinations of input. 
+We see that when side-information is included in the input for CSGCN-ADJ it does not change the performance much compared to excluding it.
+Another interesting thing to note is how the model performs worse when context is used in the input, even though we are doing context-specific recommendations.
+This could be because the CSGCN-ADJ model does not actually model context in way that fits the definition of context defined in \Cref{subsec:define_context_sideinfo}.
```

Comment:
```suggestion
This could be because the CSGCN-ADJ model does not actually model context in a way that fits the definition of context defined in \Cref{subsec:define_context_sideinfo}.
```